common/app/agents/DeeplyReadAgent.scala

```diff
@@ -33,40 +33,54 @@ class DeeplyReadAgent(contentApiClient: ContentApiClient, ophanApi: OphanApi) ex
-                  .item(path)
-                  .showTags("all")
-                  .showFields("all")
-                  .showReferences("none")
-                  .showAtoms("none")
-                val trailFromCapiResponse = contentApiClient
-                  .getResponse(capiRequest)
-                  .map { res =>
-                    res.content.flatMap { capiData =>
-                      log.info(s"Retrieved CAPI data for Deeply Read item: ${path}")
-                      deeplyReadUrlToTrail(capiData)
+        ophanApi
+          .getDeeplyRead(edition)
+          .flatMap {
+            ophanDeeplyReadItems =>
+              log.info(s"Fetched ${ophanDeeplyReadItems.size} Deeply Read items for ${edition.displayName}")
+              val constructedTrail: Seq[Future[Option[Trail]]] = ophanDeeplyReadItems.map {
+                ophanItem =>
+                  log.info(s"CAPI lookup for Ophan deeply read item: ${ophanItem.toString}")
+                  val path = removeStartingSlash(ophanItem.path)
+                  log.info(s"CAPI Lookup for path: $path")
+                  val capiRequest = contentApiClient
+                    .item(path)
+                    .showTags("all")
+                    .showFields("all")
+                    .showReferences("none")
+                    .showAtoms("none")
+
+                  contentApiClient
+                    .getResponse(capiRequest)
+                    .map { res =>
+                      res.content.flatMap { capiData =>
+                        log.info(s"Retrieved CAPI data for Deeply Read item: ${path}")
+                        deeplyReadUrlToTrail(capiData)
+                      }
+                    }
+                    .recover {
+                      case NonFatal(e) =>
+                        log.error(s"Error retrieving CAPI data for Deeply Read item: ${path}. ${e.getMessage}")
+                        None
                     }
-                  }
-                  .recover {
-                    case NonFatal(e) =>
-                      log.error(s"Error retrieving CAPI data for Deeply Read item: ${path}. ${e.getMessage}")
-                      None
-                  }
-                trailFromCapiResponse.map(_.get)
-            }
-            Future.sequence(constructedTrail)
-        }
+              }
+              Future
+                .sequence(constructedTrail)
+                .map { maybeTrails =>
+                  (edition, maybeTrails.flatten)
+                }
+
+          }
+          .recover { e =>
+            log.error(s"Failed to fetch Deeply Read items for ${edition.displayName}. ${e.getMessage()}")
+            (edition, Seq.empty)
+          }
       })
       .map(trailsList => {
-        val map = Edition.allEditions.zip(trailsList).toMap
+        val map = trailsList.toMap
+        for {
+          (edition, list) <- map
+        } yield log.info(s"Deeply Read in ${edition.displayName}: ${list.map(_.url).toString()}")
```

Comment:
This is working nicely:

<img width="544" alt="image" src="https://github.com/guardian/frontend/assets/76776/9db2cead-7dfd-4217-a241-1cf663791f45">